source/Point.h

```diff
@@ -31,11 +31,15 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 class Point {
 public:
 	Point() noexcept;
+
 	Point(double x, double y) noexcept;
 
 	// Check if the point is anything but (0, 0).
 	explicit operator bool() const noexcept;
 	bool operator!() const noexcept;
+	bool operator==(const Point &other) const noexcept;
+	bool operator!=(const Point &other) const noexcept;
+	bool operator<(const Point &other) const noexcept;
 
 	// No comparison operators are provided because I never expect to use them
```

Comment:
mb change that comment (and take it into account mb; it is unsafe tbf but then again it prob doesnt matter too much for us here, but it could for other cases)